hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/util/FutureMessage.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.mesos.hdfs.util;
+
+import org.apache.mesos.Protos;
+import org.apache.mesos.SchedulerDriver;
+import org.apache.mesos.hdfs.scheduler.Scheduler;
+
+/**
```

Comment:
I don't agree with creating a class just to execute one line of code that's called in one place. To make it testable we can create a separate method as needed, but not a separate class.